packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -266,6 +268,8 @@ class WebReleaseBundle extends Target {
         environment.outputDir.childFile(globals.fs.path.basename(outputFile.path)).path
       );
     }
+    generateVersionFile(FlutterProject.current(),environment);
```

Comment:
is this getting added to the output dependencies? Also, you need a test that this rule is doing the right thing

Reply:
yes. it is getting added to the output dependancies.